src/main/java/com/mathworks/ci/RunMatlabTestsStep.java

```diff
@@ -142,6 +153,11 @@ private String getInputArgs() {
         args.put("SimulinkTestResults", getTestResultsSimulinkTest());
         args.put("CoberturaCodeCoverage", getCodeCoverageCobertura());
         args.put("CoberturaModelCoverage", getModelCoverageCobertura());
+        if(getSourceFolder() != null && getSourceFolder().size() != 0){
```

Comment:
since fixNull is used, do we need the null check?

Reply:
Fixed